### ISSUE TITLE
Textual citations with numeric style

### ISSUE THREAD

**Issue by ashgillman** [andras-simonyi/citeproc-el#154]
Apologies in advance: I'm not sure if this is a citeproc or oc-csl.el issue.

In org, if I export with a numeric style (I've tried Nature and IEEE), and use a textual citation, I just get the number out twice.

``` org
#+cite_export: csl ~/Zotero/styles/nature.csl
#+bibliography: ~/references.bib
[cite/t:@mcnameeConsiderationOptimalTime2009] sought to identify the optimal acquisition window for SUVr with respect to correlation with DVR.
```
Gives:
> 7 7 sought to identify the optimal acquisition window for SUVr with respect to correlation with DVR.

Or for IEEE:
> [7] [7] sought to identify the optimal acquisition window for SUVr with respect to correlation with DVR.

Using the default style, or Zotero's APA style works as expected:
> McNamee et al. ([2009](file:///Users/gil2a4/Library/Mobile%20Documents/iCloud~com~logseq~logseq/Documents/org/roam/topic/20240208205420-acquisition_frame_harmonisation.html#citeproc_bib_item_5)) sought to identify the optimal acquisition window for SUVr with respect to correlation with DVR.

I have confirmed this issue with html, LaTeX and pandoc export.

**Comment by andras-simonyi** [comment]
Thanks for reporting, I'll  push a fix for the problem shortly.

**Comment by andras-simonyi** [comment]
I've merged now a branch (#157)  which should fix the problem.

**Comment by andras-simonyi** [comment]
In the absence of objections I'm going to close this issue shortly.
